docs/victoriatraces/querying/README.md

```diff
@@ -129,6 +130,29 @@ Here's a response example:
 ```
 
+#### Querying dependencies
```

Comment:
add a link to jaeger doc, like this one https://www.jaegertracing.io/docs/2.10/architecture/apis/#service-dependencies-graph